src/components/ProductList/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function ProductList({ values }) {
+  return (
+    <div>
+      {values && (
+        <span>
+          <div>
+            {values.docs.map((doc) => (
+              <div key={doc.id}>
```

Comment:
Curious here why you've nested multiple divs and a span, would you be up for clarifying the intention here?

Reply:
Well, we only refactored a little the code (but we didn't change something maybe also we should improve the code) Thanks for your question and review! We'll improve it 👍  .

Reply:
**_Changes applied!_** 😄